app/models/grda_warehouse/hud/project.rb

```diff
@@ -66,7 +68,7 @@ class Project < Base
     PROJECT_TYPE_TITLES = PROJECT_GROUP_TITLES.select { |k, _| k.in?([:ph, :es, :th, :sh, :so]) }.freeze
     HOMELESS_TYPE_TITLES = PROJECT_TYPE_TITLES.except(:ph)
     CHRONIC_TYPE_TITLES = PROJECT_TYPE_TITLES.except(:ph)
-    RESIDENTIAL_TYPE_TITLES = PROJECT_GROUP_TITLES.select { |k, _| k.in?([:ph, :es, :th, :sh, :so, :rrh, :psh]) }.freeze
+    RESIDENTIAL_TYPE_TITLES = PROJECT_GROUP_TITLES.select { |k, _| k.in?([:ph, :es, :th, :sh, :so, :rrh, :psh, :oph, :ophs]) }.freeze
```

Comment:
You added OPHS here, but not in the other places? Is this type 10 (PH – Housing with Services (no disability required for entry))?

Reply:
Thanks for catching that.  I started adding it, but removed it because it was already included in `:psh`.  Guess I missed one.